terra-jupyter-gatk/gatk-tf.patch

```diff
@@ -0,0 +1,31 @@
+--- /opt/conda/lib/python3.7/site-packages/vqsr_cnn/vqsr_cnn/models.py  2021-09-23 22:44:09.000000000 +0000
++++ models.py   2021-09-23 22:48:06.197472317 +0000
+@@ -5,7 +5,7 @@
+ from keras import layers
+ from keras import metrics
+ import keras.backend as K
+-from keras.optimizers import Adam
++from keras.optimizers import adam_v2 as Adam
```

Comment:
does it make sense to PR in gatk repo directly?

Reply:
I'm not sure. This is a hotfix for the TF upgrade issue. We can suggest this to the GATK maintainers. I assume they are fixing this in the next GATK release!